packages/core/src/utils/set-batch-item.ts

```diff
@@ -0,0 +1,5 @@
+export type SetBatchItem = {
+  key: string | Uint8Array;
+  value: string | Uint8Array;
+  ttl: number;
+};
```

Comment:
specified a new type that requires ttl if users are going to pass in per-item ttl, but this could be optional instead if we want

Reply:
i like it! I do like the idea of making it optional though.